src/main/java/com/redhat/sast/api/service/JobBatchService.java

```diff
@@ -89,7 +96,7 @@ public void executeBatchProcessing(
             LOGGER.debug(
                     "Batch {}: Found {} jobs to process. Starting sequential processing.", batchId, jobDtos.size());
 
-            processJobs(batchId, jobDtos);
+            processJobs(batchId, jobDtos, submittedBy);
```

Comment:
This is not a good approach. You are passing the submittedBy variable to this method just to set the value to every dto object. This could have been done before this method, and not introduce a new parameter here.